### ISSUE TITLE
What is the purpose of this repo?

### ISSUE THREAD

**Issue by BlueGoliath** [NVIDIA/nvapi#2]
As far as I can tell, it's just a continuation of the Open Source releases but with a much more restrictive license.

**Comment by Saancreed** [comment]
> As far as I can tell, it's just a continuation of the Open Source releases but with a much more restrictive license.

No, this is the standard SDK that predates Open Source one, for when you actually want to use NVAPI in your application and not reimplement it from scratch. The content of this repo is the same as what's been available for a long time at https://developer.nvidia.com/gameworksdownload#?search=nvapi except it's easier to use in other projects like [Path Tracing SDK now that it can be included like any other git submodule](https://github.com/NVIDIAGameWorks/Path-Tracing-SDK/blob/58642a2c920fa4db5a6a56a62097b222b5700980/.gitmodules#L17-L19) instead of forcing build systems or package managers to download it separately.
